UnitsNet.Tests/CustomCode/AmountOfSubstanceTests.cs

```diff
@@ -58,5 +69,46 @@ public void NumberOfParticlesInTwoMolesIsDoubleAvogadroConstant()
             var numberOfParticles = twoMoles.NumberOfParticles();
             Assert.Equal(AmountOfSubstance.AvogadroConstant * 2, numberOfParticles);
         }
+
+        [Fact]
+        public void TenMolesOfOxygenToMassEqualToExpected()
+        {
+            AmountOfSubstance tenMoles = AmountOfSubstance.FromMoles(10);
+            MolarMass molarMass = MolarMass.FromGramsPerMole(MolarMassOfOxygen);
+            AssertEx.EqualTolerance(10 * MolarMassOfOxygen, (tenMoles * molarMass).Grams, MolesTolerance);
```

Comment:
These tests are good, but I struggle a bit to see what is actually tested here vs what is setup code.
Please restructure the tests on the format
```c#
// Arrange
/* setup code here */

// Act
/* ideally a single line calling a single method/operator returning a single result */

// Assert
/* Assert statements here */
```


I usually omit the `// Arrange` comment, because I find that obvious. I also sometimes omit `// Act` and `// Assert` comments if there are three clearly defined paragraphs of Arrange-Act-Assert code.

Reference: http://wiki.c2.com/?ArrangeActAssert

Once restructured, see if the naming of the method can be improved. We have no hard guideline here, but we try to follow the naming convention of `WhatIsTested_WhatAreTheInputs_WhatIsTheResult()`, such as `Multiply_Given10MolesAndOxygenMolarMass_Returns10MolarMassOfOxygen`. You can probably see from my example name that I didn't fully understand what was tested here, so please choose your own wording.

Another naming style I find just as functional is `VolumeTimesMassConcentrationEqualsMass` as you have used below. It works.

Reply:
Most tests renamed to follow something like the second convention: SomeResultFromSomethingAndSomethingElse